arches/app/templates/views/components/widgets/datepicker.htm

```diff
@@ -5,14 +5,16 @@
 {% block form %}
 <div class="row widget-wrapper">
     <div class="form-group">
-        <label class="control-labx mesel widget-input-label" for="" data-bind="text:label"></label>
+        <span class="control-labx mesel widget-input-label" for="" data-bind="text:label"></span>
         <!-- ko if: node -->
         <i data-bind="css: {'ion-asterisk widget-label-required': node.isrequired}"></i>
         <!-- /ko -->
         <div class="col-xs-12">
-            <div style="pad-hor">
+            <div class="pad-hor">
```

Comment:
becase style="pad-hor" didn't apply any valid style, we shouldn't make this into a class. We should just remove these references altogether.  Should just be a plain div.